### PR TITLE
Add metro@3.0.13 with git autoupdate

### PR DESCRIPTION
**Pull request by Piicksarn**
**git repo url:** https://github.com/olton/Metro-UI-CSS/tree/3.0.13
**Watch 606
  Star 4589
 Fork 1720**
@Amomo could you help me check out the PR for #6260  ? thank you~
- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity 
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

**Comment by Amomo** [comment]
@Piicksarn don't we need `fonts/` folder?

File: ajax/libs/metro/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  ],
+  "homepage": "http://metroui.org.ua",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com:olton/Metro-UI-CSS.git"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/bfintal/Counter-Up.git",
```
Comment (Amomo) [inline ajax/libs/metro/package.json]: You set a wrong target!

Comment (PeterDaveHello) [inline ajax/libs/metro/package.json]: nice catch :+1:

**Comment by Amomo** [comment]
@Piicksarn sorry I didn't notice that you have updated the PR.
Could you rebase it with latest branch again? Thank you! :-D

**Comment by Piicksarn** [comment]
@Amomo, would you mind to have a look at the pr again?
thank you!

File: ajax/libs/metro/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/olton/Metro-UI-CSS.git",
+    "basePath": "build",
+    "files": [
+      "js/metro*.js",
+      "css/metro*.css",
+      "fonts/metro*"
```
Comment (PeterDaveHello) [inline ajax/libs/metro/package.json]: So why not just `**/metro*`?

**Comment by Piicksarn** [comment]
@Amomo, would you mind to have a look at the pr again?
thank you!

**Pull request merged**